modules/ROOT/attachments/admin-api.yaml

```diff
@@ -584,6 +584,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/partitions_local_summary'
+  /partitions/majority_lost:
+    get: 
+      tags: 
+      - Partitions
+      summary: Get partitions with lost majority
+      description: List of partitions with lost majority given an input set of dead nodes.
+      operationId: majority_lost
+      # responses: 
+        # 200:
+          # content: 
+            # application/json:
+      parameters:
+        - name: dead_nodes
+          in: query
+          required: true
+          schema:
+            type: string   
+  /partitions/force_recover_from_nodes:
+    post: 
+      tags: 
+      - Partitions
+      summary: Force recover partitions
+      description: Force recover partitions from input list of nodes. 
```

Comment:
This endpoint requires an input list but no parameters defined in [spec](https://github.com/redpanda-data/redpanda/blob/dev/src/v/redpanda/admin/api-doc/partition.json#L393)?

Reply:
Here is [the request body schema](https://github.com/redpanda-data/redpanda/blob/e9b93b9cd397862145e33a7e193b118c85495c57/src/v/redpanda/admin/partition.cc#L1019)